daemon.c

```diff
@@ -251,6 +252,7 @@ static void parse_options(int argc, char **argv)
 		printf("\t-m <port>   port to use for MOUNT service\n");
 		printf
 		    ("\t-t          TCP only, do not listen on UDP ports\n");
+		printf("\t-4          open sockets only with IPv4\n");
```

Comment:
Asi bych to napsal jako "Use IPv4-only sockets". Nebo možná by bylo lepší "Listen only on IPv4" (nevím, co přesně to má dělat).